### PR TITLE
add gcp tech note

### PR DESCRIPTION
**Pull request by leena-jawale**

### CODE REVIEW

File: stories/topics/tech-note-gcp-extensionnodes.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[id="tech-note-gcp-extensionnodes"]
+
+= Extension Nodes
+
+{AAPonGCP} can be extended by adding an extension nodes after you have purchased and launched an extension from public or private offer.
+
+*Main Deployment Name* is a required field when you deploy an extension node from marketplace.
```
Comment (ianf77) [inline stories/topics/tech-note-gcp-extensionnodes.adoc:7]: Perhaps say where it's a required field? Maybe a cross reference to the relevant section?

**Review by ianf77 — COMMENTED** [review]
one comment

**Review by zkayyali812 — APPROVED** [review]

**Comment by leena-jawale** [comment]
@ianf77 I don't know how ref works here. Do we have to create a new adoc file for ref. I want to add like this 

```
*Main Deployment Name* is a required field when you deploy an extension node from marketplace, See xref:proc-gcp-deploying-extension-nodes[Extension Node]
```

Do I have to create a new ref file `ref-gcp-deploying-extension-nodes.adoc` with same content as of `proc-gcp-deploying-extension-nodes.adoc`

**Comment by ianf77** [comment]
No. The format is xref:<id of the modules you want to refer to>[Title of the module you want to refer to] so as long as `proc-gcp-deploying-extension-nodes.adoc`  exists then you have the reference correct.

File: stories/topics/tech-note-gcp-extensionnodes.adoc [hunk 2]
```diff
@@ -4,6 +4,6 @@
 
 {AAPonGCP} can be extended by adding an extension nodes after you have purchased and launched an extension from public or private offer.
 
-*Main Deployment Name* is a required field when you deploy an extension node from marketplace. 
+*Main Deployment Name* is a required field when you deploy an extension node from marketplace, See xref:proc-gcp-deploying-extension-nodes[Extension Node]
```
Comment (ianf77) [inline stories/topics/tech-note-gcp-extensionnodes.adoc:7]: [Deploying an extension node]

**Review by ianf77 — COMMENTED** [review]
One comment.

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**